lib/cli/commands/getinfo.ts

```diff
@@ -1,11 +1,128 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { GetInfoRequest } from '../../proto/xudrpc_pb';
+import Table, { VerticalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { GetInfoRequest, GetInfoResponse, LndInfo, RaidenInfo } from '../../proto/xudrpc_pb';
+
+type generalInfo = {
+  version: string;
+  numPeers: number;
+  numPairs: number;
+  nodePubKey: string;
+  orders: {own: number, peer: number} | undefined
+};
+
+const displayChannels = (channels: any, asset: string) => {
+  const table = new Table() as VerticalTable;
+  Object.keys(channels).forEach((key: any) => {
+    table.push({
+      [colors.blue(key)] : channels[key],
+    });
+  });
+  console.log(colors.underline(colors.bold(`\nLnd ${asset} channels:`)));
+  console.log(table.toString(), '\n');
+};
+
+const displayChainsList = (list: any[], asset: string) => {
+  const table = new Table() as VerticalTable;
+  list.forEach((asset, i) => {
+    if (asset) {
+      table.push({ [colors.blue(`${i + 1}.`)]: `${asset.chain}-${asset.network}` });
+    }
+  });
+  if (table.length !== 0) {
+    console.log(colors.underline(colors.bold(`\nLnd ${asset} chains:`)));
+    console.log(table.toString(), '\n');
+  }
+};
+
+const displayUriList = (uris: string[], asset: string) => {
+  const table = new Table() as VerticalTable;
+  uris.forEach((uri, i) => table.push({ [`${i + 1}.`]: uri }));
+  console.log(colors.underline(colors.bold(`\nLnd ${asset} uris:`)));
+  console.log(table.toString(), '\n');
+};
+
+const displayLndInfo = (asset: string, info: LndInfo.AsObject) => {
+  const basicInfotable = new Table() as VerticalTable;
+  basicInfotable.push(
+    { [colors.blue('Error')]: info.error },
+  );
+  if (info.blockheight) {
+    basicInfotable.push({ [colors.blue('Block Height')]: info.blockheight });
+  }
+  if (info.version) {
+    basicInfotable.push({ [colors.blue('Version')]: info.version });
+  }
+  if (info.alias) {
+    basicInfotable.push({ [colors.blue('Alias')] : info.alias });
+  }
+
+  console.log(colors.underline(colors.bold(`\nLnd ${asset} info:`)));
+  console.log(basicInfotable.toString(), '\n');
+
+  if (info.channels) {
+    displayChannels(info.channels, asset);
+  }
+
+  if (!info.error) {
+    displayChainsList(info.chainsList, asset);
+  }
+
+  if (info.urisList.length > 0) {
+    displayUriList(info.urisList, asset);
+  }
+};
+
+const displayGeneral = (info: generalInfo) => {
+  const table = new Table() as VerticalTable;
+  table.push(
+    { [colors.blue('Version')]: info.version },
+    { [colors.blue('Pairs')]: info.numPairs },
+    { [colors.blue('Peers')]: info.numPeers },
+    { [colors.blue('Public key')]: info.nodePubKey },
+  );
+  if (info.orders) {
+    table.push(
+      { [colors.red('Own orders')]: info.orders.own },
```

Comment:
Personally I find the red and green here a bit jarring in contrast with the other blue labels, thoughts on making these all blue @kilrau?

Reply:
yeah, let's make these all blue